diff --git a/node_modules/detect-libc/lib/detect-libc.js b/node_modules/detect-libc/lib/detect-libc.js
index fe49987..bc72ad0 100644
--- a/node_modules/detect-libc/lib/detect-libc.js
+++ b/node_modules/detect-libc/lib/detect-libc.js
@@ -3,6 +3,8 @@
 
 'use strict';
 
+console.log("patched detect-libc");
+
 const childProcess = require('child_process');
 const { isLinux, getReport } = require('./process');
 const { LDD_PATH, readFile, readFileSync } = require('./filesystem');
